packages/storybook/src/stories/ino-radio-group/ino-radio-group.stories.ts

```diff
@@ -13,37 +13,35 @@ export default {
       handles: ['checkedChange ino-radio'],
     },
   },
-  decorators: [
-    story => decorateStoryWithClass(story),
-    story => {
-      useEffect(() => {
-        const radioGrp = document.querySelector('#radio-grp');
-        const eventHandler = e =>
-          radioGrp.setAttribute('value', e.target.getAttribute('value'));
-        radioGrp.addEventListener('checkedChange', eventHandler);
-        const evtHandler = e => {
-          radioGrp.setAttribute('value', e.detail);
-        };
-        radioGrp.addEventListener('valueChange', evtHandler);
-        return () => {
-          radioGrp.removeEventListener('checkedChange', eventHandler);
-          radioGrp.removeEventListener('valueChange', evtHandler);
-        };
-      });
-      return story();
-    },
-  ],
+  decorators: [ story => decorateStoryWithClass(story) ],
   args: {
     value: 'opt-2',
-    alignments: 'horizontal',
+    alignment: 'horizontal',
   },
 } as Meta<Components.InoRadioGroup>;
 
+const updateValueOnChange = (id) => {
+  useEffect(() => {
+    const radioGrp = document.querySelector(id);
+    const eventHandler = e =>
+      radioGrp.setAttribute('value', e.target.getAttribute('value'));
+    radioGrp.addEventListener('checkedChange', eventHandler);
+    const evtHandler = e => {
+      radioGrp.setAttribute('value', e.detail);
+    };
+    radioGrp.addEventListener('valueChange', evtHandler);
+    return () => {
+      radioGrp.removeEventListener('checkedChange', eventHandler);
+      radioGrp.removeEventListener('valueChange', evtHandler);
+    };
+  });
+}
+
 const template = new TemplateGenerator<Components.InoRadioGroup>(
```

Comment:
The template formatting is a little bit off. The opening and closing tags of the `div` should be aligned. You can fix that be indenting everything from line 41 to 52.

<img width="673" alt="image" src="https://github.com/inovex/elements/assets/103122411/2f347684-9e65-4c82-82ca-f271db68a426">